.gitignore

```diff
@@ -1,7 +1,7 @@
 _site
 _built
 
-TAGS
+# TAGS
```

Comment:
@jgerigmeyer this came from a yarn/rollup commit, but it ends up ignoring our tag pages. Feel free to delete or replace with something more targeted?

Reply:
This is a holdover from old projects, not added intentionally. Just remove it.